internal/cli/metrics_opt.go

```diff
@@ -37,3 +41,17 @@ func (opts *MetricsOpts) NewProcessMetricsListOptions() *atlas.ProcessMeasuremen
 
 	return o
 }
+
+// ValidatePeriodStartEnd validates period, start and end flags.
+func (opts *MetricsOpts) ValidatePeriodStartEnd() error {
+	if opts.Period == "" && opts.Start == "" && opts.End == "" {
+		return fmt.Errorf("either the --period flag or the --start and --end flags are required")
```

Comment:
if you don't need to format the string you can do 
```suggestion
		return errors.New("either the --period flag or the --start and --end flags are required")
```

That said, we sometimes try to use the flag constants in case the unlikely event the flag name changes
```go
		return fmt.Errorf("either the --%s flag or the --%s and --%s flags are required", flags.Period, flags.Start, flags.End)
```

(I'm not sure those are the actual name of the flags)

Reply:
Thanks, I'll change that.